src/components/ResourceBrowser/ResourceList/ResourceList.tsx

```diff
@@ -264,7 +265,14 @@ export default function ResourceList() {
                 otherEvents.push(iterator)
             }
         }
-        return [...warningEvents, ...otherEvents]
+        return [
+            ...warningEvents.sort((a, b) => {
+                return k8sStyledAgeToSeconds(a['last seen']) - k8sStyledAgeToSeconds(b['last seen'])
+            }),
+            ...otherEvents.sort((a, b) => {
+                return k8sStyledAgeToSeconds(a['last seen']) - k8sStyledAgeToSeconds(b['last seen'])
```

Comment:
Please create a const key for 'last seen'

Reply:
Also, can we create an eventAgeComparator itself?

Reply:
done